SRC/classes/AccountInsertRecord.cls

```diff
@@ -0,0 +1,30 @@
+/**
+*  Description     :  Insert Record IN Account
+*
+*  Created By      :  Kamlesh Kumar
+*
+*  Created Date    :  07/04/2022
+*
+*  Revision Logs   :    V1.0 - Created - Kamlesh Kumar
+**/
+public with sharing class AccountInsertRecord {
+    /**
+    *   @description    :  Insert Record IN Account
+    *
+    *   @return         :   void
+    *
+    *   @revision Log   :   V1.1 - Created  - 07/04/2022 - Developer Name - Kamlesh Kumar
+    **/
+    public AccountInsertRecord() {
+        Account acc = new Account();
+        acc.id ='0015i00000BDPHrAAP';
```

Comment:
Remove the hard-coded value and use the SOQL query to retrieve the record and then pass the record Id